README.md

```diff
@@ -7,6 +7,76 @@ The Afterpay Android SDK makes it quick and easy to provide an excellent payment
 ## Documentation
 Documentation for usage can be found [here][docs], including the [getting started][docs-getting-started] guide and [UI component][docs-ui] docs.
 
+## V3 Documentation
+
+### Launching the Checkout
```

Comment:
Since all of the docs got moved offsite during v3 work, this got added here. I'll move these offsite along with the other documentation